### PR TITLE
Fix scripts

### PR DESCRIPTION
**Pull request by deepfire**
Issues: #376

Note, that this doesn't touch the benchmarking-specific scripts, which could be converted to using the shared generated genesis.

### CODE REVIEW

File: scripts/trace-acceptor.sh [hunk 1]
```diff
@@ -9,6 +11,7 @@ ${CMD} \
     --topology "./configuration/simple-topology.json" \
     --database-path "./db/" \
     --genesis-file "configuration/mainnet-genesis.json" \
+    --genesis-hash "5f20df933584822601f9e3f8c024eb5eb252fe8cefb24d1317dc3d432e940ebb" \
```
Comment (Jimbo4350) [inline scripts/trace-acceptor.sh]: Shouldn't the genesis hash be pulled from `lib-node.sh`?

File: scripts/lib-node.sh [hunk 2]
```diff
@@ -1,7 +1,15 @@
-
-genesis_hash=`cat configuration/GenesisFiles/GENHASH`
-genesis_root="configuration/GenesisFiles"
+genesis_root="$(realpath configuration/genesis)"
 genesis_file="${genesis_root}/genesis.json"
+
```
Comment (Jimbo4350) [inline scripts/lib-node.sh]: 👍

File: scripts/trace-acceptor.sh (hunk 1 above)
Comment (deepfire) [reply scripts/trace-acceptor.sh]: @Jimbo4350, it's hard-coded to mainnet here, so no it can't be pulled from there.

**Pull request merged**